cmd/app/app_test.go

```diff
@@ -155,6 +156,42 @@ func TestListDevicesWithGui(t *testing.T) {
 	assert.Empty(t, out)
 }
 
+func TestMaxCores(t *testing.T) {
+	out, err := runApp(
+		&Flags{LogFile: "/dev/null", MaxCores: 1},
+		[]string{},
+		true,
+		testdev.DevicesInfoGetterMock{},
+	)
+
+	assert.Contains(t, out, "set to 1")
```

Comment:
It would be nice to also check return value of `runtime.GOMAXPROCS(0)` in all the new tests.

Reply:
done